templates/index.html

```diff
@@ -1,692 +1,427 @@
-                snap developers to describe the ways their snaps should be
-                connected. A snap can be connected to the host system OS
-                using plugs, too, which is how it can access shared user
-                documents or other services like OpenGL.</p>
-
-                <p>Snaps can share files with other snaps from the same vendor,
-                or with community-maintained shared snaps which act as libraries
-                of common data or code, for
-                example using the content-sharing interface. This reduces duplication while still ensuring rigorous
-                security and update control.</p>
-
-                <p>When you install a snap, the system will automatically connect
-                up the standard, safe plugs and slots. For example, your new game
-                will automatically get access to the OpenGL libraries on your
-                system because that’s considered normal behaviour for a game.
-                But there may be other plugs and slots you can connect up by
-                choice, and  snaps will document those individually.</p>
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/13929d8e-brackets.png" alt="" />
+              <h5 class="u-no-margin"><a href="/brackets">Brackets</a></h5>
             </div>
-            <div id="snapcraft_home_how-snaps-work_architecture" class="eight-col">
-                <h3>The architecture of a snappy system</h3>
-                <p>So a snappy system has a collection of snaps, each of which
-                is read-only, each of which has its own set of independent
-                writable directories, and each of which can talk to other snaps
-                and the host OS through a set of plugs and slots with matching
-                interfaces.</p>
-
-                <p>
-                    <img class="diagram--wide" src="https://assets.ubuntu.com/v1/143dff64-Update-Snapcraft%28dot%29io-architecture_v2.svg" />
-                </p>
-
-                <p>The snap system will present all the snap apps under <code class="command-inline">/snap/bin/</code>,
-                so with that on your <code class="command-inline">$PATH</code> you can easily run snap commands. It
-                will also integrate all the snap daemons (services) with your
-                init system so that they are automatically started and stopped
-                as needed.</p>
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/4eca1f4e-discord.png" alt="" />
+              <h5 class="u-no-margin"><a href="/atom">Atom</a></h5>
```

Comment:
Here it should be Discord

Reply:
Could we also get some x2 images for hdpi?